<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientJSON</class>
 <widget class="QMainWindow" name="ClientJSON">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientJSON</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushbutton_Connect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Подключиться</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_EnterIP">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_EnterPort">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>4007</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_IP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Port">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Порт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AddItem">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DeleteItem">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_ConnectionStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Lamps">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LampOn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Включить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LampOff">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Выключить</string>
    </property>
   </widget>
   <zorder>pushButton_DeleteItem</zorder>
   <zorder>pushbutton_Connect</zorder>
   <zorder>lineEdit_EnterIP</zorder>
   <zorder>lineEdit_EnterPort</zorder>
   <zorder>label_IP</zorder>
   <zorder>label_Port</zorder>
   <zorder>pushButton_AddItem</zorder>
   <zorder>textBrowser_ConnectionStatus</zorder>
   <zorder>comboBox_Lamps</zorder>
   <zorder>pushButton_LampOn</zorder>
   <zorder>pushButton_LampOff</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
